sdk/python/tests/integration/materialization/test_lambda.py

```diff
@@ -33,7 +33,10 @@ def test_lambda_materialization_consistency():
         },
         registry_location=RegistryLocation.S3,
     )
-    lambda_environment = construct_test_environment(lambda_config, None)
+    # TODO(adchia): figure out why entity_key_serialization_version 2 breaks with this test
+    lambda_environment = construct_test_environment(
+        lambda_config, None, entity_key_serialization_version=1
+    )
```

Comment:
This is probably becase we need to build and push a new docker image (that uses the new version number). The docker image isn't automatically updated via CI at the moment.